api/cpp/esp-idf/slint/src/slint-esp.cpp

```diff
@@ -230,12 +230,14 @@ void EspPlatform::run_event_loop()
                         std::swap(buffer1, buffer2.value());
                     } else {
                         for (int y = o.y; y < o.y + s.height; y++) {
+#if BSP_LCD_COLOR_SPACE == ESP_LCD_COLOR_SPACE_BGR
```

Comment:
We'll have to check if this still works on the S3 box  (that this is properly defined at this location.
(I'm not entirely sure how this gets defined if we don't include any bsp header)

Reply:
I was curious about this, too, and added an #error inside this #ifdef and did builds for s3-usb-otb/s2-kaluga and s3-box, and it triggered for all of them. I'll double check to see where exactly it comes from.

Reply:
maybe this is equivalent to 0 == 0  if none are defined?

Reply:
ESP_LCD_COLOR_SPACE_BGR is an enum value:
https://github.com/espressif/esp-idf/blob/60a2bf6a68867fd468a250966230a067f8d97ea2/components/esp_lcd/include/esp_lcd_types.h#L50
So i think this should somehow be a runtime check and not a compile time check.
btw, i think i looked for a way back in the days to check that but couldn't find a proper way. I think you'll need to pass an option to the init function of something like that.